jobs/integr8ly/ocp4/ocp4-testing-executor.yaml

```diff
@@ -0,0 +1,225 @@
+      - string:
+          name: CLUSTER_DOMAIN
+          default: 'open.redhat.com'
+          description: "Cluster domain - open.redhat.com or example.opentlc.com for RHPDS clusters and intlyqe.com for clusters created using openshift-install"
+      - bool:
+          name: PDS_CLUSTER
+          default: true
+          description: "Indicates whether cluster was provisioned using RHPDS or openshift-install"
+      - bool:
+          name: SELF_SIGNED_CERTS
+          default: true
+          description: "Indicates whether cluster uses self signed certificates or not"
+      - string:
+          name: ADMIN_USERNAME
+          default: 'opentlc-mgr'
+          description: "Admin username for the OCP4 RHPDS cluster. For openshift-install clusters kubeadmin will be used by default"
+      - string:
+          name: ADMIN_PASSWORD
+          default: ''
+          description: "Jenkins Credentials for RHPDS admin user"
+      - string:
+          name: INTEGREATLY_OPERATOR_REPOSITORY
+          default: 'https://github.com/integr8ly/integreatly-operator.git'
+          description: "Repository of the Integreatly Operator"
+      - string:
+          name: INTEGREATLY_OPERATOR_BRANCH
+          default: 'master'
+          description: "Branch of the Integreatly Operator repository"
+      - string:
+          name: REGISTRY_NAMESPACE
+          default: 'integreatly'
+          description: "Registry name in quay.io"
+      - choice:
+          name: INSTALLATION_TYPE
+          description: "Installation type (workshop for PDS clusters, managed otherwise"
+          choices:
+            - workshop
+            - managed
+      - string:
+          name: NAMESPACE_PREFIX
+          description: "Prefix of Integreatly Operator namespace, defauls to `BUILD_NUMBER`"
+      - bool:
+          name: CLEANUP_BEFOREHAND
+          default: true
+          description: "Indicates whether to cleanup before execution"
+      - bool:
+          name: CLEANUP_AFTERWARD
+          default: true
+          description: "Indicates whether to cleanup after execution"
+      - string:
+          name: NAMESPACE_CLEANUP_BEFOREHAND
+          description: "Namespace to be removed before the Integreatly Operator deploy. Can be left empty"
+      - string:
+          name: GH_CLIENT_ID
+          description: "GitHub Client ID for OAuth Apps, required for some of the walkthroughs"
+      - string:
+          name: GH_CLIENT_SECRET
+          description: "GitHub Client Secret for OAuth Apps, required for some of the walkthroughs"
+      - string:
+          name: TEST_SUITES_REPOSITORY
+          default: 'https://gitlab.cee.redhat.com/integreatly-qe/integreatly-qe.git'
+          description: "Repository of Integreatly test suites, YAML templates are stored there"
+      - string:
+          name: TEST_SUITES_BRANCH
+          default: 'development'
```

Comment:
Should we go with master? Since there's plan to stop using development.

Reply:
hmm. yes we probably should